src/icons/ItemIcon.tsx

```diff
@@ -93,23 +94,23 @@ const ItemIcon: FC<ItemIconProps> = ({
     case ItemType.LOCAL_FILE:
     case ItemType.S3_FILE: {
       if (mimetype) {
-        if (MIME_TYPES.IMAGE.includes(mimetype)) {
+        if (MIME_TYPES.isImage(mimetype)) {
```

Comment:
Nitpick: not sure if it makes sense that `MIME_TYPES` is uppercase, it seems particularly striking here on a method call. I see it more as a static object, so I would probably write it as `MimeTypes`. Should this be changed in SDK?

Reply:
Yes we can change the naming in sdk.